src/app/rpmostree-dbus-helpers.c

```diff
@@ -905,14 +905,27 @@ rpmostree_print_signatures (GVariant *variant,
                                                    OSTREE_GPG_SIGNATURE_FORMAT_DEFAULT);
       else
         {
-          gboolean valid;
-          g_variant_get_child (v, OSTREE_GPG_SIGNATURE_ATTR_VALID, "b", &valid);
-          const char *fingerprint;
-          g_variant_get_child (v, OSTREE_GPG_SIGNATURE_ATTR_FINGERPRINT, "&s", &fingerprint);
           if (i != 0)
             g_string_append (sigs_buffer, sep);
-          g_string_append_printf (sigs_buffer, "%s signature by %s\n", valid ? "Valid" : "Invalid",
-                                  fingerprint);
+
+          gboolean isKeyMissing;
+          g_variant_get_child (v, OSTREE_GPG_SIGNATURE_ATTR_KEY_MISSING, "b", &isKeyMissing);
+
+          const char *fingerprint;
+          g_variant_get_child (v, OSTREE_GPG_SIGNATURE_ATTR_FINGERPRINT, "&s", &fingerprint);
+
+          if(isKeyMissing)
```

Comment:
Our code style is [snake case](https://en.wikipedia.org/wiki/Snake_case) for variables, also space between identifier and paren.  So: `if (is_key_missing)`

Reply:
Whops, my fault. I'm coming from the .NET world initially. I'll fix that later today.